From: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
Date: Mon, 11 Feb 2019 11:52:38 +0100
Subject: [PATCH] crypto: user: remove crypto_cfg_mutex
Origin: https://www.kernel.org/pub/linux/kernel/projects/rt/5.0/older/patches-5.0.3-rt1.tar.xz

crypto_cfg_mutex was never used since it got introduced in commit

  cac5818c25d04 ("crypto: user - Implement a generic crypto statistics")

Cc: Corentin Labbe <clabbe@baylibre.com>
Signed-off-by: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
---
 crypto/crypto_user_stat.c |    2 --
 1 file changed, 2 deletions(-)

--- a/crypto/crypto_user_stat.c
+++ b/crypto/crypto_user_stat.c
@@ -20,8 +20,6 @@
 
 #define null_terminated(x)	(strnlen(x, sizeof(x)) < sizeof(x))
 
-static DEFINE_MUTEX(crypto_cfg_mutex);
-
 extern struct sock *crypto_nlsk;
 
 struct crypto_dump_info {
